### DISCUSSION TITLE
Auto Colored Chat Text

### DISCUSSION THREAD

**Discussion by DippinDot** [runelite/runelite#15769] [Ideas]
Hey I was wondering if its possible to make a plugin that automatically changes your overhead text chat color or if there are any restrictions that Jagex has placed that prevent this from being feasible.

 I imagine it would be simple if you have menu entry swapper on, you need to press enter anyway to chat; it would probably be easy on pressing enter to have a plugin that automatically types green: or greenwave: or any of the other variants.

**Comment by Enriath** [comment]
Autotyping is against the rules. 

Changing overhead text colour locally would be fine, though I'm not entirely sure the API supports it.
